lib/new_relic/agent/samplers/delayed_job_sampler.rb

```diff
@@ -97,11 +100,11 @@ def queue_counts(column_name)
           # here that's valid on 2.x through 4.1, so split it up.
           result = if ::ActiveRecord::VERSION::MAJOR.to_i < 4
             ::Delayed::Job.count(:group => column_name,
-                                 :conditions => [QUEUE_QUERY_CONDITION, now])
+              :conditions => [QUEUE_QUERY_CONDITION, now])
           else
             ::Delayed::Job.where(QUEUE_QUERY_CONDITION, now).
-                           group(column_name).
-                           count
+              group(column_name).
+              count
```

Comment:
@amhuntsman  - Is this better? Or would you prefer `count` indented once more?

Reply:
This is fine. It just makes it clearer that it's a continuation of line 105.